File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int i=0;
    int h=0;
    int x=0;
    int y=0;

    do
    {
        printf("Height:");
        h = GetInt();
    } while (h<0 || h>23);

    for (i=0; i<h; i++)
    {
        for (x=1; x<h-i; x++)
        {
            printf(" ");
        }
        for (y=0; y<i+2; y++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario.py
import cs50

def main():
    while True:
        print("height: ", end = "")
        height = cs50.get_int()
        if height >= 0 and height <= 23:
            break
        
    for i in range(height):
        print (" "*(height - i - 1)+ "#"*(i+2)+ "  "+ "#"*(i+2))
            
if __name__ == "__main__":
    main()